webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
+  CleanWebpackPlugin = require('clean-webpack-plugin'),
+  HtmlWebpackPlugin = require('html-webpack-plugin'),
+  ExtractTextPlugin = require('extract-text-webpack-plugin');
+
+const extractPlugin = new ExtractTextPlugin({filename: './assets/css/app.css'});
+
+const config = {
+
+  context: path.resolve(__dirname, 'src'),
+
```

Comment:
I'm sceptic about this newline:
If there is ruleset, that after every webpack config key there is a newline, than ommit this, but imho it's redundant.
I'll leave it for discussion, and wont write comment in each key